RevenueCatUI/View+PresentPaywallFooter.swift

```diff
@@ -19,6 +19,7 @@ import SwiftUI
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
 extension View {
 
+    // swiftlint:disable line_length
```

Comment:
Hmm I might be wrong but I think we need to reenable this check after the affected block, so we still check the rest of the file?

Reply:
Agreed... I think this has to be enabled ☝️